package.json

```
@@ -0,0 +1,60 @@
+{
+  "name": "@asyncapi/optimizer",
+  "version": "0.0.1",
+  "description": "This library will optimize the AsyncAPI specification file.",
+  "main": "./lib/index.js",
+  "types": "./lib/index.d.ts",
+  "scripts": {
+    "test": "echo \"No test specified\"",
+    "build": "tsc",
+    "lint": "eslint --max-warnings 0 --config .eslintrc .",
+    "get:version": "echo $npm_package_version",
+    "get:name": "echo $npm_package_name",
+    "generate:assets": "npm run build",
+    "bump:version": "npm --no-git-tag-version --allow-same-version version $VERSION",
+    "release": "semantic-release"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/asyncapi/optimizer.git"
+  },
+  "author": "Khuda Dad Nomani",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/asyncapi/optimizer/issues"
+  },
+  "homepage": "https://github.com/asyncapi/optimizer#readme",
+  "devDependencies": {
+    "@semantic-release/commit-analyzer": "^8.0.1",
+    "@semantic-release/github": "^7.2.3",
+    "@semantic-release/npm": "^7.1.3",
+    "@types/jest": "^26.0.23",
+    "@types/node": "^15.12.1",
```

Please add packages related to eslint and eslint plugins:

```suggestion
    "@types/node": "^15.12.1",
    "@typescript-eslint/eslint-plugin": "^4.26.0",
    "@typescript-eslint/parser": "^4.26.0",
    "eslint": "^7.28.0",
    "eslint-plugin-github": "^4.1.3",
    "eslint-plugin-security": "^1.4.0",
    "eslint-plugin-sonarjs": "^0.8.0-125",
```